slices/fontconfig-config.yaml

```diff
@@ -0,0 +1,67 @@
+package: fontconfig-config
+
+slices:
+  config:
+    essential:
+      - fonts-dejavu-core_config
+      - fonts-dejavu-core_fonts
+    contents:
+      /etc/fonts/conf.avail/10-antialias.conf:
```

Comment:
I prefer to avoid using globs. I understand the simplification, but at the same time, it feels like an abuse of globs. There's value in having the paths specified so that you know precisely what's being installed. 

I'd rather only use globs for simplifying boilerplate paths, architectures and minor versions, eg. `/usr/lib/*-linux-*/libunwind.so.1.*:`

but if there's a strong case for it, I can change it

Reply:
well you just gave a perfect example of why we shouldn't use globs just to reduce the number of lines in the SDF :)

if we add `fonts-dejavu-extra`, this `fonts` slice will immediately conflict with it.

Reply:
decided to keep this practice for when we're dealing with common paths that work as a drop in for multiple packages